docs/example_static_html.md

```diff
@@ -3,50 +3,190 @@ title: "Example: Plain Old Static HTML"
 layout: docs
 ---
 
-Maybe you want to try Tilt but don't have a suitable project on hand.
+The best indicator of a healthy development workflow is a short feedback loop.
 
-That's OK! Let's look at a project that just serves static HTML.
+Kubernetes is a huge wrench in the works.
 
-These projects can also be useful to confirm Tilt is working as expected in your environment.
+Let's fix this.
 
-## oneup
+In this example, we're going to take you through a very simple shell script that
+serves static HTML.
 
-Oneup is a simple app that uses busybox to serve HTML files.
+We'll use Tilt to:
 
-First, check out the Tilt repo.
+- Run the server on Kubernetes
+- Measure how long it takes to deploy
+- Optimize the updates for fast feedback.
```

Comment:
"the updates" is vague. Maybe like, "optimize the propagation of local changes up to the running server" or something?

Reply:
channeling DB here but: I don't think "deploy" is the word we want. "Measure how long it take for local edits to propagate to the server"?

...I'm being verbose cuz I want to be precise, and like I mention below, "updates" is a liiiiittle vague/overloaded... but. hmm. "Measure how long it takes to update our server with new code"? "Measure how long it takes to see new code running in the cluster"?